### ISSUE TITLE
ofxclient fails during import on non-unix or on App Engine

### ISSUE THREAD

**Issue by sneak** [captin411/ofxclient#2]
``````
  File "/base/data/home/apps/s~eeqjdev/dev.366103472990174732/eeqj/geckoboard.py", line 2, in <module>
    import ofxclient
  File "/base/data/home/apps/s~eeqjdev/dev.366103472990174732/thirdparty/ofxclient/__init__.py", line 4, in <module>
    from config import OfxConfig
  File "/base/data/home/apps/s~eeqjdev/dev.366103472990174732/thirdparty/ofxclient/config.py", line 11, in <module>
    DEFAULT_CONFIG = os.path.expanduser('~/ofxclient.ini')
  File "/python27_runtime/python27_dist/lib/python2.7/posixpath.py", line 259, in expanduser
    import pwd
ImportError: No module named pwd```
``````

Basically, trying to use the password database module (pwd) on non-unix will fail, or, in my case, on Google App Engine.  It's not safe to assume that os.path.expanduser() is safe to use for portability
